packages/terra-clinical-result/src/proptypes/statusPropTypes.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types';
+
+/**
+ * Enum for possible Result Statuses.
+ * values mapped from http //hl7.org/fhir/2018Sep/valueset-observation-status.html
```

Comment:
Missing the colon after http too

Reply:
Resolved here: https://github.com/cerner/terra-clinical/pull/574/commits/a946b3b3eac5a238fba7c3584c69237aa5ecaa50